src/compounds/RequestStats/RequestStats.jsx

```diff
@@ -3,42 +3,35 @@ import { Card } from 'react-bootstrap'
 import PropTypes from 'prop-types'
 import './request-stats.css'
 
-const RequestStats = ({ addClass, billingInfo, createdAt, deadline, shippingInfo }) => {
-  const shippingAddress = shippingInfo.address.split(/\n/)
-  const billingAddress = billingInfo.address.split(/\n/)
-
-  return (
-    <Card className='request-stats w-25'>
-      <Card.Header className={`${addClass}`}>
-        <Card.Title className='mb-0'>Request Info</Card.Title>
-      </Card.Header>
-      <Card.Body>
-        <Card.Subtitle>Date of request</Card.Subtitle>
-        <Card.Text className='mb-4'>{createdAt}</Card.Text>
-        <Card.Subtitle>Proposals required by</Card.Subtitle>
-        <Card.Text className='mb-4'>{deadline}</Card.Text>
-        <Card.Subtitle>Shipping Address</Card.Subtitle>
-        {shippingAddress.map((line, index) => (
-          <Card.Text
-            key={shippingInfo.id}
-            className={`${(index === shippingAddress.length - 1) ? 'mb-4' : 'mb-0'}`}
-          >
-            {line}
-          </Card.Text>
-        ))}
-        <Card.Subtitle>Billing Address</Card.Subtitle>
-        {billingAddress.map((line, index) => (
-          <Card.Text
-            key={billingInfo.id}
-            className={`${(index === billingAddress.length - 1) ? '' : 'mb-0'}`}
-          >
-            {line}
-          </Card.Text>
-        ))}
-      </Card.Body>
-    </Card>
-  )
-}
+const RequestStats = ({ addClass, billingInfo, createdAt, deadline, shippingInfo }) => (
+  <Card className='request-stats w-25'>
+    <Card.Header className={`${addClass}`}>
+      <Card.Title className='mb-0'>Request Info</Card.Title>
+    </Card.Header>
+    <Card.Body>
+      <Card.Subtitle>Date of request</Card.Subtitle>
+      <Card.Text className='mb-4'>{createdAt}</Card.Text>
+      <Card.Subtitle>Proposals required by</Card.Subtitle>
+      <Card.Text className='mb-4'>{deadline}</Card.Text>
+      <Card.Subtitle>Shipping Address</Card.Subtitle>
+      <Card.Text
+        as='address'
+        className='mb-4 request-stats-address'
+        key={shippingInfo.id}
+      >
+        {shippingInfo.address}
+      </Card.Text>
```

Comment:
no longer mapping over multiple paragraphs. also, render the element as an `address`.

same for the billing info below.